app/mailers/notifier.rb

```diff
@@ -4,29 +4,36 @@
 class Notifier < ApplicationMailer
   def password_reset_instructions(user, mission: nil)
     build_reset_url(user)
-    mail(to: user.email, reply_to: reply_to(mission),
+    mail(to: user.email, reply_to: coordinator_emails(mission),
          subject: t("notifier.password_reset_instructions"))
   end
 
   def intro(user, mission: nil)
     @user = user
     build_reset_url(user)
-    mail(to: user.email, reply_to: reply_to(mission),
+    mail(to: user.email, reply_to: coordinator_emails(mission),
          subject: t("notifier.welcome", site: Settings.site_name))
   end
 
   def sms_token_change_alert(mission)
+    raise ArgumentError, "Mission must not be nil" unless mission
+
     @mission = mission
     @site_name = Settings.site_name
-    mail(to: reply_to(mission), reply_to: reply_to(mission),
+    all_emails = (coordinator_emails(mission) + admin_emails).uniq
+    mail(to: all_emails, reply_to: all_emails,
          subject: t("notifier.sms_token_change.subject", mission_name: mission.name))
   end
 
   private
 
-  def reply_to(mission)
+  def coordinator_emails(mission)
     return [] if mission.nil?
-    User.with_roles(mission, :coordinator).pluck(:email).uniq[0, 10] # Max of 10 reply to, should be rare.
+    User.with_roles(mission, :coordinator).pluck(:email).uniq[0, 10] # Max of 10, should be rare.
+  end
+
+  def admin_emails
+    User.where(admin: true).pluck(:email).uniq[0, 10] # Max of 10, should be rare.
```

Comment:
Hmm... let's not limit admin emails. Seems potentially dangerous and confusing. I'm on the fence about coordinator ones, the likelihood of there being many coordinators seems higher, and they're going into the reply-to header of a commonly sent email...